swift_algorithms/Controllers/AboutViewController.swift

```diff
@@ -4,7 +4,7 @@ import Anchorage
 final class AboutViewController: UIViewController {
     
     override var preferredStatusBarStyle: UIStatusBarStyle {
-        return .lightContent
+        return .default
     }
```

Comment:
@rickwierenga We should probably switch on the theme here, right?

Reply:
Right. I hadn't seen that. I added the responsive solution to the other controllers.

I tried creating an extension to UIViewController and set the statusbar style from there. This is a better solution in my opinion. However, that code doesn't get called. I have no clue why. (this change isn't committed)